Description: fix Unicode issues in PygmentsBridge.highlight_block()
 In some circumstances pygments.highlight() returns a byte string instead of
 Unicode string. PygmentsBridge.highlight_block() does expect Unicode strings
 only, so convert such strings forcibly to Unicode.
Bug-Debian: http://bugs.debian.org/660930
Bug: https://bitbucket.org/birkenfeld/sphinx/issue/905/typeerror-expected-a-character-buffer
Author: Anton Gladky <gladky.anton@gmail.com>
Last-Update: 2012-04-02

--- a/sphinx/highlighting.py
+++ b/sphinx/highlighting.py
@@ -204,6 +204,7 @@
         try:
             formatter = self.get_formatter(**kwargs)
             hlsource = highlight(source, lexer, formatter)
+            hlsource = unicode(hlsource)
             if self.dest == 'html':
                 return hlsource
             else:
